packages/typespec-client-generator-core/src/interfaces.ts

```diff
@@ -604,3 +606,105 @@ export enum UsageFlags {
   // Set when model is used in conjunction with an application/json content type.
   Json = 1 << 8,
 }
+
+interface SdkExampleBase {
+  kind: string;
+  name: string;
+  description: string;
+  filePath: string;
+  rawExample: any;
+}
+
+export interface SdkHttpOperationExample extends SdkExampleBase {
+  kind: "http";
+  parameters: SdkHttpParameterExample[];
+  responses: Map<number, SdkHttpResponseExample>;
+}
+
+export interface SdkHttpParameterExample {
+  parameter: SdkHttpParameter;
+  value: SdkTypeExample;
+}
+
+export interface SdkHttpResponseExample {
+  response: SdkHttpResponse;
+  headers: SdkHttpResponseHeaderExample[];
+  value?: SdkTypeExample;
```

Comment:
Is the value here body example value?

Reply:
want to keep consistent for all other example type, value is represent the example value.

Reply:
sync offline, will refine the interface.